driver-adapters/neon-http-lambda-basic/index.js

```diff
@@ -40,6 +40,29 @@ exports.handler = async () => {
     //     },
     //   ],
     // }),
+    // createManyAndReturn: await prisma.user.createManyAndReturn({
```

Comment:
I imagine this is commented out because http mode but it would be nice to have a comment here about it

Reply:
It's the same comment as above normally, so I did not add anything
But it looks like the previous block does not have a comment for this test here, it's a bit of a mess these tests